modules/odc-verifying-the-status-of-your-service-binding-from-the-topology-view.adoc

```diff
@@ -0,0 +1,32 @@
+:_content-type: PROCEDURE
+[id="odc-verifying-the-status-of-your-service-binding-from-the-topology-view_{context}"]
+== Verifying the status of your service binding from the Topology view
+
+The *Developer* perspective helps you verify the status of your service binding through the *Topology* view.
+
+.Procedure
+To verify the status of your service binding, consider the following approaches:
+
+* Successful service binding
++
+When a service binding is successful, click on the binding connector. A side panel appears displaying the *Connected* status under the *Details* tab. 
```

Comment:
Please apply the following IBM Style Guide guideline by replacing the 7 instances of "click on" in this PR with "click":

"Avoid using a phrasal verb (a verb plus a preposition) if the verb alone provides the same meaning." "click on > click"

Reply:
Done.